enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/context.tsx

```diff
@@ -170,7 +173,7 @@ export const InteractiveQuestionProvider = ({
         isQueryRunning,
         resetQuestion,
         onReset: onReset || resetQuestion,
-        onNavigateBack: onNavigateBack || resetQuestion,
+        onNavigateBack: onNavigateBack || undefined,
```

Comment:
is this because `onNavigateBack` can return null, so we coalesce it to undefined?

Reply:
Not exactly - the `InteractiveQuestionResult` is used in the independent `InteractiveQuestion` and `InteractiveQuestionAdHoc`, which is the interactive question component that's used when accessing a question within an interactive dashboard. `onNavigateBack` is only needed for the second usage, when we need to return to the dashboard. 

All that is to say, if `onNavigateBack` isn't passed as a prop, we shouldn't display the back button and we shouldn't give it any functionality

Reply:
@oisincoveney I see, I wonder if we could simply return `onNavigateBack` then, without falling back to `undefined` if `onNavigateBack` already can be undefined

```suggestion
        onNavigateBack,
```

Reply:
yeah that makes sense!